synapse/handlers/space_summary.py

```diff
@@ -67,6 +69,12 @@ def __init__(self, hs: "HomeServer"):
         self._server_name = hs.hostname
         self._federation_client = hs.get_federation_client()
 
+        # TODO Allow for multiple workers to share this data.
+        # TODO Expire pagination tokens.
+        self._pagination_sessions: Dict[
+            str, Tuple[Deque[_RoomQueueEntry], Set[str]]
+        ] = {}
```

Comment:
could do with a comment describing the contents? (Or possibly a proper attr.s class rather than a tuple?)